array-methods-more/find-by-gender.js

```diff
@@ -1,7 +1,14 @@
 /*
   Refactor this function to use .find and ES6 syntax
  */
-
+const findByGender = (persons, gender)=>{
```

Comment:
I think the question meant to use .filter method.
Your solution is correct but does not take full advantage of ES6!
Check out my solution:
```javascript
const findByGender = (persons, gender)=>{
  const oneGenderArray = persons.filter((person) => {
    return person.gender===gender
  });
  return oneGenderArray;
};
```